Initial macos support

outdegree opened the pull request.

GreatAttractor commented on README.md:
````diff
@@ -302,6 +302,27 @@ $ cpack
 ```
 Note that building needs to be performed in an environment corresponding to the selected target system, so that proper shared objects are linked to (“environment” = a full system installation, a Docker image, or similar).
 
+#### 11.1.3. Building for macOS
````
Please add 11.1.3 also to the Table of Contents at the beginning.

GreatAttractor commented on src/about.cpp:
```diff
@@ -159,8 +162,11 @@ class c_AboutDialog: public wxDialog
 
 void c_AboutDialog::OnLibrariesClick(wxCommandEvent&)
 {
+    // glGetString() segfaults on macOS. I guess GL Context is not initialized.
```
You're right, but this `#if` is needed anyway (in case OpenGL back end is disabled in `config.in`, there's a build error here). So I think we don't need this comment.

outdegree replied:
The comment should have gone under the #if expression as it is irrelevant to the expression but a crash I have not resolved yet. I can remove the comment anyway and have a bug tix open instead.

GreatAttractor replied:
Yes, please do that (remove the comment, but open a bug). Now that I thought about it, the bug seems to be due to a) having OpenGL support built-in, b) launching ImPPG in CPU mode, so at no point was an OpenGL context initialized/made current, c) attempting to call a `gl*` function. Under Windows and Linux c) is allowed (just returns an empty/null string), but not on macOS.

GreatAttractor commented on README.md:
```diff
@@ -302,6 +302,27 @@ $ cpack
+
+To build ImPPG for macOS you will need Xcode and [Homebrew](https://brew.sh) installed.
+
+OpenMP is not supported yet as Apple has disabled OpenMP in clang toolchain distributed with Xcode. At this writing the build method has been verified on macOS Big Sur 11.5.2 and Xcode 12.5.1.
```
To save the users checking of Git history, this is better:

> As of 9/2021, (...)

GreatAttractor commented:
The PR looks good; use "ready for review" to advance from "draft" status, I'll merge it then (unless you have some additional changes to make).

outdegree commented:
> The PR looks good; use "ready for review" to advance from "draft" status, I'll merge it then (unless you have some additional changes to make).

I moved from draft state and added Polish README changes. Please do a 2nd pass to make sure I did not mess up anything.

GreatAttractor commented on README_pl.md:
```diff
@@ -309,6 +310,29 @@ $ cpack
+
+Do budowy ImPPG pod macOS wymagana jest instalacja Xcode oraz [Homebrew](https://brew.sh).
+
+OpenMP nie jest obecnie używany ponieważ Apple wyłączył OpenMP w kompilatorze clang dystrybuowanym razem z Xcode. Obecna metoda budowy została przetestowana we wrzesniu 2021 na macOS Big Sur 11.6 oraz Xcode 12.5.1.
```
A few minor corrections:

>, ponieważ

> wrześniu

> używając

> Linuksie
